public/css/main.css

```diff
@@ -20,14 +19,96 @@ a {
 }
 
 /*** GENERAL ***/
-
 body {
   background-color: #F2F2F2;
 }
 
 /*** HEADER BAR ***/
+.header {
+  background-color: #409A93;
+  font-family: 'Noto Sans', sans-serif;
+  padding: 2.5rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.naz-events{
+  color: white;
+  font-weight: bold;
+  font-size: 2.3rem;
+}
+
+.calendar {
+  font-size: 2.3rem;
+  color: white;
+  margin-top: 0.1rem;
+}
+
+.plus{
+  margin-top: 0.2rem;
+  font-size: 2.3rem;
+  color: white;
+}
+
+.languageNav {
```

Comment:
For consistency I think we should just use kebab case for classes - see the 'design principles' part of the issue [here](https://github.com/foundersandcoders/nazareth-events/issues/71)